cumulusci/core/config.py

```diff
@@ -802,11 +802,15 @@ def create_org(self):
         if self.devhub:
             devhub = ' --targetdevhubusername {}'.format(self.devhub)
 
+        namespaced = ''
```

Comment:
Would be nicer as 1-liner:
`namespaced = '' if self.namespaced else ' -n'`

Reply:
I find that format hard to read.  It's similar to [x.y for x in foo] vs a normal for loop.  Perhaps that's just that I commonly forget the syntax for it.  I'd personally prefer to leave it this way as we use this pattern all over the place in cumulusci already but I'm open to discussion on that.

Reply:
I like doing it this way but certainly won't block the PR for it.